### PR TITLE
#22 Home Page Member Profile Widget

### PR DESCRIPTION
**Pull request by chenhe-henry**
## Rationale
Display Code.Sydney members at the homepage, as an overview of members' profiles. 

## Advice for Reviewers & Testing Notes
- **Open one terminal**
- ```cd client```
- ``` npm install``` to make sure node-sass, sass-loader, axios, font awesome, and json-server work properly.
- ``` npm run serve```

- **Open another terminal**
- ```cd client```
- run ```json-server --watch membersDb.json``` to watch local json file for axios
- Go to your browser and navigate to ```localhost:8080```

- Note: Save members' data in membersDb.json temporarily. Need run ```json-server --watch membersDb.json``` to fetch data. The baseURL for axios will be replaced when the backend is ready.

## Screenshots:

- Add a screenshot or two of your changes, where applicable.

image
image
image

## Task Name

- #22 Homepage-member profile widget

## Linting Checklist
- [x] No commented code
- [x] Code Formatted nicely (Prettier)
- [x] PR your own code before you assign reviewers

### CODE REVIEW

**Comment by promie** [comment]
image


On mobile view the members photo do not look quite aligned.....

File: client/src/components/MemberPreview.vue [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        this.members = res.data;
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }
+};
+</script>
+
+<style lang="scss" scoped>
```
Comment (promie) [inline client/src/components/MemberPreview.vue:71]: Impressive job with sass man.

**Comment by chenhe-henry** [comment]
> image
> 
> On mobile view the members photo do not look quite aligned.....

fixed by changing the responsive font-size~

**Comment by promie** [comment]
image

Still not quite aligned. The font of the title looks so small. Btw, You got two Linkedin links under the mentor's profile. Is this intentional? I mean would they be replaced with another social media logo down the track?

**Comment by chenhe-henry** [comment]
> Still not quite aligned. The font of the title looks so small. Btw, You got two Linkedin links under the mentor's profile. Is this intentional? I mean would they be replaced with another social media logo down the track?

I will adjust the font-size later.  Yes, the two Linkedin icon is just to test the display to make sure badge icons and social links' icons can display nice, it will be replaced later with the v-for loop of real back-end data. And will double-check the align issue, thanks for your comment, Promie :)

**Comment by promie** [comment]
> Great work @chenhe-henry!
> 
> I have a couple styling quibbles, but I won't waste your time with that right now because we're still figuring out how to handle styling for the project so we'd probably just end up making you change it again in a month.
> 
> Can you please add the instructions for running the json-server to the README.md?
> After that, I'm happy to approve your changes.

I second this comment.

**Comment by QjStout** [comment]
@chenhe-henry, savour that sweet sweet feeling of hitting the merge button friend

**Comment by engramar** [comment]
Congratulations on getting your first PR merged @chenhe-henry Apologies if I was not able to review your changes, very busy recently. Congrats again!

**Pull request merged**